cli/global.go

```diff
@@ -42,17 +43,24 @@ func ConfigureGlobals(app *kingpin.Application) {
 		OverrideDefaultFromEnvar("AWS_VAULT_PROMPT").
 		EnumVar(&GlobalFlags.PromptDriver, promptsAvailable...)
 
+	app.Flag("use-login-keychain", "Uses your default login keychain rather a new one").
+		OverrideDefaultFromEnvar("AWS_VAULT_USE_LOGIN_KEYCHAIN").
+		BoolVar(&GlobalFlags.UseLoginKeychain)
+
 	app.PreAction(func(c *kingpin.ParseContext) (err error) {
+		var keyringName = DefaultKeyringName
+		if GlobalFlags.UseLoginKeychain && GlobalFlags.Backend == keyring.KeychainBackend {
+			keyringName = "login"
```

Comment:
Is `login` what we want here? Seems like it's possible for the "default" keychain to be named differently

Reply:
Yeah, fair point. I will investigate that. Perhaps "default" keychain is the phrasing we should be using.

Reply:
Looks a bit like you can use 'NULL' to mean 'default keychain':

https://developer.apple.com/documentation/security/1393225-seckeychainitemcreatefromcontent#topic-content